code/datums/mapgen/planetary/WasteGenerator.dm

```diff
@@ -66,9 +66,9 @@
 		BIOME_COLD_CAVE = list(
 			BIOME_LOWEST_HUMIDITY = /datum/biome/cave/waste,
 			BIOME_LOW_HUMIDITY = /datum/biome/cave/waste/rad,
-			BIOME_MEDIUM_HUMIDITY = /datum/biome/cave/waste,
-			BIOME_HIGH_HUMIDITY = /datum/biome/cave/waste/rad,
-			BIOME_HIGHEST_HUMIDITY = /datum/biome/cave/waste
+			BIOME_MEDIUM_HUMIDITY = /datum/biome/cave/waste/conc,
+			BIOME_HIGH_HUMIDITY = /datum/biome/cave/waste/conc,
+			BIOME_HIGHEST_HUMIDITY = /datum/biome/cave/waste/conc
```

Comment:
I dont really feel like this needs to be 3/5 of the cave.. I would have it be (waste waste rad conc conc)